docker-compose.yml

```diff
@@ -3,6 +3,7 @@ services:
   base:
      volumes:
        - ./:/var/lib/eventkit/
+       - ../django-audit-logging:/var/lib/.virtualenvs/eventkit/src/django-audit-logging
```

Comment:
This is for development.   To include changes we should be tagging a new version of the django-audit-logger in the requirements.txt.